BCB 546 R.Rmd

```diff
@@ -187,17 +186,20 @@ for (i in 1:10) {
 filtered_fangSNP <- filter(merged_fangSNP , Position != "multiple",  Position != "unknown")
 
 ggplot(data = filtered_fangSNP) +
-  geom_bar(mapping = aes(x = Chromosome)) +
+  geom_bar(mapping = aes(x = as.factor(as.double(Chromosome)))) + # as.double sorts chr from 1:10; as.factor transform each chr into a factor 
```

Comment:
That is actually makes graph much better and neat! Thank you for all the suggestions